utils/ansible-runner-service.sh

```diff
@@ -416,7 +416,7 @@ check_access() {
 manage_ansible_hosts_file() {
     echo "Linking the runner service inventory to ceph-ansible hosts"
     # create hosts file if it doesn't exist
-    [ ! -f $CEPH_ANSIBLE_HOSTS ] && touch $CEPH_ANSIBLE_HOSTS
+    [ ! -s $CEPH_ANSIBLE_HOSTS ] && touch $CEPH_ANSIBLE_HOSTS
 
     # Set up a new hosts file if the hosts file contains hosts details already or it is an INI file
     if grep -q -e 'hosts:' -e ']' "$CEPH_ANSIBLE_HOSTS" 2>/dev/null; then
```

Comment:
this is where I think the -s test applies...i.e if the file exists and is not empty, we save it - otherwise it's empty and ready to use. As I said earlier an inventory doesn't have to contain groups [group] blocks, it could just be a list of servers - this is why I'm suggested, if the file exists and contains anything at all...save it, tell the user, and set up a new one for the install.